Marlin/configuration_store.cpp

```diff
@@ -187,20 +187,21 @@ void Config_StoreSettings()  {
   EEPROM_WRITE_VAR(i, planner.max_e_jerk);
   EEPROM_WRITE_VAR(i, home_offset);
 
+  uint8_t dummy_uint8 = 0;
   uint8_t mesh_num_x = 3;
   uint8_t mesh_num_y = 3;
   #if ENABLED(MESH_BED_LEVELING)
     // Compile time test that sizeof(mbl.z_values) is as expected
     typedef char c_assert[(sizeof(mbl.z_values) == (MESH_NUM_X_POINTS) * (MESH_NUM_Y_POINTS) * sizeof(dummy)) ? 1 : -1];
     mesh_num_x = MESH_NUM_X_POINTS;
     mesh_num_y = MESH_NUM_Y_POINTS;
-    EEPROM_WRITE_VAR(i, mbl.active);
+    dummy_uint8 = mbl.status & 0x01; // Do not save 'is active'
```

Comment:
`_BV` doesn't cost anything if you decide to use bit indices. Alternatively you could use…
```cpp
enum MBLBits { MBL_ACTIVE_BIT = 0x01, MBL_HAS_MESH_BIT = 0x02 };
. . .
dummy_uint8 = mbl.status & MBL_ACTIVE_BIT;
```